### PR TITLE
Improve CLI help pages

### PR DESCRIPTION
**Pull request by AuHau**
Improved CLI help pages with updated synopsis and information about logging support in IPFS.

### CODE REVIEW

File: core/commands/root.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ const (
 var Root = &cmds.Command{
 	Helptext: cmdkit.HelpText{
 		Tagline:  "Global p2p merkle-dag filesystem.",
-		Synopsis: "ipfs [--config=<config> | -c] [--debug=<debug> | -D] [--help=<help>] [-h=<h>] [--local=<local> | -L] [--api=<api>] <command> ...",
+		Synopsis: "ipfs [--config=<config> | -c] [--debug | -D] [--help] [-h] [--local | -L] [--api=<api>] [--offline] [--cid-base=<base>] [--upgrade-cidv0-in-output] [--enc=<encoding> | --encoding=<encoding>] [--stream-channels] [--timeout=<timeout>] <command> ...",
```
Comment (Kubuxu) [inline core/commands/root.go]:
```suggestion
		Synopsis: "ipfs [--config=<config> | -c] [--debug | -D] [--help] [-h] [--local | -L] [--api=<api>] [--offline] [--cid-base=<base>] [--upgrade-cidv0-in-output] [--encoding=<encoding> | --enc ] [--stream-channels] [--timeout=<timeout>] <command> ...",
```

File: core/commands/root.go [hunk 2]
```diff
@@ -72,6 +72,7 @@ TOOL COMMANDS
   update        Download and apply go-ipfs updates
   commands      List all available commands
   cid           Convert and discover properties of CIDs
+  log			Manage and show logs of running daemon
```
Comment (Kubuxu) [inline core/commands/root.go]: Indentation doesn't match

**Comment by Kubuxu** [comment]
Thanks! I've left some comments.

File: core/commands/root.go (hunk 1 above)
Comment (AuHau) [reply core/commands/root.go]: Thanks for the tweak

File: core/commands/root.go (hunk 2 above)
Comment (AuHau) [reply core/commands/root.go]: Damned tabulators, not sure why I used them 😅

**Comment by AuHau** [comment]
You are welcome!
All fixed.

File: core/commands/root.go [hunk 3]
```diff
@@ -29,7 +29,7 @@ const (
 var Root = &cmds.Command{
 	Helptext: cmdkit.HelpText{
 		Tagline:  "Global p2p merkle-dag filesystem.",
-		Synopsis: "ipfs [--config=<config> | -c] [--debug | -D] [--help] [-h] [--local | -L] [--api=<api>] [--offline] [--cid-base=<base>] [--upgrade-cidv0-in-output] [--encoding=<encoding> | --enc] [--stream-channels] [--timeout=<timeout>] <command> ...",
```
Comment (Stebalien) [inline core/commands/root.go]: @Kubuxu LGTY?

File: core/commands/root.go [hunk 4]
```diff
@@ -29,7 +29,7 @@ const (
...
+		Synopsis: "ipfs [--config=<config> | -c] [--debug | -D] [--help] [-h] [--api=<api>] [--offline] [--cid-base=<base>] [--upgrade-cidv0-in-output] [--encoding=<encoding> | --enc] [--timeout=<timeout>] <command> ...",
```
Comment (Stebalien) [inline core/commands/root.go]: @Kubuxu LGTY?

Comment (Kubuxu) [reply core/commands/root.go]: Yeah, `--stream-channels` is an internal option isn't it?

**Pull request merged**